internal/handlers/telegram/handler.go

```diff
@@ -26,3 +26,17 @@ func messageHandler(tg *Client, u tgbotapi.Update) {
 
 	tg.sendToIrc(formatted)
 }
+
+/*
+documentHandler once a document received, uses information about the document
+to format a message to return to the client.
+*/
+func documentHandler(tg *Client, u tgbotapi.Update) {
+	header := tg.Settings.Prefix + u.Message.From.UserName +
+		tg.Settings.Suffix
+
+	message := " shared a file (" + u.Message.Document.MimeType +
+		") on Telegram with title" + "'" + u.Message.Document.FileName + "'."
```

Comment:
I would generally want the caption a user adds to their document, as this would give IRC users more background information as to what the file is. It may be worthwhile sending over the filename instead of a caption if the user does not supply one.

Reply:
Additionally, line 39 does not have a space after "with title...", and so the filename is sent over without any spaces.

Reply:
Good eye for the space. I solved it. As of the caption, I used an if-else statement to check if there is a caption. I use the caption if it is not an empty string, and the filename if the caption is empty.